src/main/clojure/clara/macros.clj

```diff
@@ -174,8 +174,10 @@
                                                      ;; that a rule wouldn't, or at least shouldn't for
                                                      ;; clarity, start the names of other locals or vars
                                                      ;; with "?".
-                                                     (mapv (comp symbol name) all-bindings))]
-              (com/compile-action all-bindings
```

Comment:
Looking at this again, i don't think `all-bindings` is required at all after these changes.

Reply:
If I understand this correctly all-bindings is being populated from the constraints after [this transformation](https://github.com/cerner/clara-rules/blob/0.19.1/src/main/clojure/clara/rules/compiler.clj#L906) and that is why this change is needed?